packages/common/src/api/GqlContext.tsx

```diff
@@ -9,23 +9,31 @@ import {
 import { AuthError } from '../authentication/AuthContext';
 import { LocalStorage } from '../localStorage';
 
+const permissionExceptions = ['reports', 'stockCounts', 'invoiceCounts'];
 interface ResponseError {
   message?: string;
   path?: string[];
   extensions?: { details?: string };
 }
 
-function hasError(errors: ResponseError[], error: AuthError) {
-  return errors.some(({ message }: { message?: string }) => message === error);
-}
+const hasError = (errors: ResponseError[], error: AuthError) =>
+  errors.some(({ message }: { message?: string }) => message === error);
+
+const hasPermissionException = (errors: ResponseError[]) =>
```

Comment:
didn't especially like adding in the exceptions - though if the check fails, it will default to showing errors.